content/core-features/authentication.textile

```diff
@@ -111,10 +113,10 @@ The @tokenParams@ argument in @authCallback@ is available for convenience, allow
 
 h3(#token-process). Ably Token is issued by your servers and passed to clients.
 
-Using our REST or Realtime client libraries, an "Ably Token is requested by your servers":/realtime/authentication#request-token from Ably and then handed to the client-side client library. The client-side client library then uses that "Ably Token":#tokens to authenticate with Ably. This is an alternative approach for authentication that allows you to issue "Ably Tokens":#tokens directly as opposed to providing Ably "@TokenRequests@":/realtime/types#token-request from your servers. The advantage for clients is it saves one round trip request as they do not need to request an "Ably Token":#tokens themselves. The disadvantage is that your servers must communicate with Ably each time an "Ably Token":#tokens is required.
+Using our REST or Realtime client libraries, an "Ably Token is requested by your servers":/realtime/authentication#request-token from Ably and then handed to the client-side client library. The client-side client library then uses that "Ably Token":#tokens to authenticate with Ably. This is an alternative approach for authentication that enables you to issue "Ably Tokens":#tokens directly as opposed to providing Ably "@TokenRequests@":/realtime/types#token-request from your servers. The advantage for clients is it saves one round trip request as they do not need to request an "Ably Token":#tokens themselves. The disadvantage is that your servers must communicate with Ably each time an "Ably Token":#tokens is required.
```

Comment:
"client-side client library" -> "Ably client library SDK"?

Reply:
Yes, that looks really odd, but I see what they are trying to say. There are two Ably Client Library SDKs, one on the client-side and one on the server-side. I will try to come up with a better way to express that.

Reply:
I've reworded to avoid the "client-side client" construct.